src/main/java/com/rocana/lucene/codec/v1/RocanaBlockTreeTermsReader.java

```diff
@@ -228,8 +250,8 @@ public BlockTreeTermsReader(PostingsReaderBase postingsReader, SegmentReadState
           throw new CorruptIndexException("invalid sumTotalTermFreq: " + sumTotalTermFreq + " sumDocFreq: " + sumDocFreq, termsIn);
         }
         final long indexStartFP = indexIn.readVLong();
-        FieldReader previous = fields.put(fieldInfo.name,       
-                                          new FieldReader(this, fieldInfo, numTerms, rootCode, sumTotalTermFreq, sumDocFreq, docCount,
+        RocanaFieldReader previous = fields.put(fieldInfo.name,       
+                                          new RocanaFieldReader(this, fieldInfo, numTerms, rootCode, sumTotalTermFreq, sumDocFreq, docCount,
```

Comment:
The spacing formatting doesn't conform to Rocana standards. Do we apply those standards to open sourced code? (I vote yes.)